HeterogeneousCore/CUDAServices/src/CUDAService.cc

```diff
@@ -439,9 +456,6 @@ struct CUDAService::Allocator {
   template <typename ...Args>
   Allocator(size_t max, Args&&... args): maxAllocation(max), deviceAllocator(args...), hostAllocator(std::forward<Args>(args)...) {}
 
-  void devicePreallocate(int numberOfDevices, const std::vector<unsigned int>& bytes);
```

Comment:
@makorte I haven't found these used or implemented anywhere

Reply:
Thanks, that's what I figured out as well after seeing you removing them. Probably an oversight in #172 to leave them around.